aten/src/ATen/native/cuda/BinaryMiscOpsKernels.cu

```diff
@@ -69,11 +69,20 @@ void xlog1py_kernel_cuda(TensorIteratorBase& iter) {
   });
 }
 
+void ldexp_kernel_cuda(TensorIteratorBase& iter) {
+  AT_DISPATCH_FLOATING_TYPES_AND2(at::ScalarType::Half, at::ScalarType::BFloat16, iter.input_dtype(0), "ldexp_cuda", [&] {
+    gpu_kernel_with_scalars(iter, []GPU_LAMBDA(scalar_t x, int exp) -> scalar_t {
```

Comment:
btw I don't think you need `gpu_kernel_with_scalars` here, because ldexp doesn't support scalar overloads, just `gpu_kernel` should be enough